src/pages/api/profile/addresses/[id].ts

```diff
@@ -96,6 +96,84 @@ const getHandler: Handler = async (request, response) => {
   );
 };
 
+const patchHandler: Handler = async (request, response) => {
+  const webErrors = {
+    invalidProfileToken: ERRORS_DATA.INVALID_PROFILE_TOKEN,
+    invalidScopes: ERRORS_DATA.INVALID_SCOPES,
+    unreachable: ERRORS_DATA.UNREACHABLE,
+    invalidQueryString: ERRORS_DATA.INVALID_QUERY_STRING,
+    addressNotFound: ERRORS_DATA.ADDRESS_NOT_FOUND,
+    invalidUserAddressPayload: ERRORS_DATA.INVALID_USER_ADDRESS_PAYLOAD,
+  };
+
+  return getTracer().span(
+    "PATCH /api/profile/addresses/[id] handler",
+    async (span) => {
+      const userCookie = await getServerSideCookies<UserCookie>(request, {
+        cookieName: "user-cookie",
+        isCookieSealed: true,
+        cookieSalt: configVariables.cookieSalt,
+      });
+
+      if (!userCookie) {
+        response.statusCode = HttpStatus.TEMPORARY_REDIRECT;
+        response.setHeader("location", "/");
+        response.end();
+        return;
+      }
+
+      const addressId = request.query.id;
```

Comment:
Isn't it in `request.params`?

Reply:
Nope, `params` doesn't exist on type `NextApiRequest`. The parameters are inside the `query` object 🙂